src/lib/interceptor/search-request.interceptor.spec.ts

```diff
@@ -66,7 +66,6 @@ describe('SearchRequestInterceptor', () => {
         it('return search request dto when input is valid', async () => {
             expect(await interceptor.validateBody({ select: ['col1', 'col2'] })).toEqual({
                 select: ['col1', 'col2'],
-                take: 20,
```

Comment:
During the validation stage, take was assigned with default values, and this was being serialized in the query(nextCursor). 
Now, assignment  of default value for take is done only in the intercept method, and only user input is serialized in the query.

If user don't specify `take`, it does not serialize into nextCursor, and assigned default value in intercept method